test/dashboard-e2e/api/api-helpers.js

```diff
@@ -7,51 +7,82 @@ export class ApiHelpers {
         this.bearerToken = bearerToken;
     }
 
-    async getTests() {
-        const request = `${this.apiUrl}/tests`;
-
+    async makeGet(requestUrl) {
```

Comment:
Optionally, you could add `this.apiUrl` directly here, like:
```suggestion
    async makeGet(requestUrl) {
        requestUrl = `${this.apiUrl.replace(/\/+$/, '')}/${requestUrl.replace(/^\/+/, '');
```

Then all usages would be like `return this.makeGet('/organizations')`